src/gregorio-utils.c

```diff
@@ -237,7 +237,7 @@ static void check_input_clobber(char *input_file_name, char *output_file_name)
             buf = gregorio_malloc(bufsize);
         }
         if (current_directory == NULL) {
-            fprintf(stderr, _("can't determine current directory"));
+            fprintf(stderr, _("can't determine current directory\n"));
```

Comment:
Should we add "error:" to the beginning of this message in order to be consistent with the others?

Reply:
Done (and for several other messages)